README.md

```diff
@@ -30,7 +30,7 @@ __ /path/to/catalogs/<catalog_name>/
 
 
 ```bash
-$ conda create -n hipscatenv
+$ conda create -n hipscatenv python=3.8
```

Comment:
Does this mean it will only work with 3.8 or is it >= 3.8?

Reply:
Based on Brigitta's comment, I've just removed these lines from the README.